src/index.html

```diff
@@ -1,13 +1,55 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="main">
+      <div class="stars__star stars--0">
+        <div class="star--card inactive"></div>
+        <div class="star--card inactive"></div>
+        <div class="star--card inactive"></div>
+        <div class="star--card inactive"></div>
+        <div class="star--card inactive"></div>
+      </div>
```

Comment:
1. I would change the name of this div's class. I think simply `stars` would be better and more clear since it is a row of stars. Not a single star.
2. Divs inside should have only one class. According to BEM it should be named stars__star